packages/esm-patient-programs-app/src/programs/programs-form.workspace.tsx

```diff
@@ -223,27 +226,19 @@ const ProgramsForm: React.FC<ProgramsFormProps> = ({
       name="enrollmentLocation"
       control={control}
       render={({ field: { onChange, value } }) => (
-        <Select
-          aria-label="enrollment location"
-          id="location"
-          labelText={t('enrollmentLocation', 'Enrollment location')}
-          onChange={(event) => onChange(event.target.value)}
-          value={value}
-        >
-          {availableLocations?.length > 0 &&
-            availableLocations.map((location) => (
-              <SelectItem key={location.uuid} text={location.display} value={location.uuid}>
-                {location.display}
-              </SelectItem>
-            ))}
-        </Select>
+        <LocationPicker
+          selectedLocationUuid={value}
+          onChange={onChange}
+          hideWelcomeMessage={true}
```

Comment:
There's no prop named `hideWelcomeMessage`

These are the available props:
https://github.com/openmrs/openmrs-esm-core/blob/53ce380c2651fd06de96f564f7e64b60495544dd/packages/framework/esm-styleguide/src/location-picker/location-picker.component.tsx#L16